tests/foreman/cli/test_import.py

```diff
@@ -151,6 +170,63 @@ def import_user_data():
     ],)
 
 
+def import_hc_data():
+    """Random data for Organization Import tests"""
+
+    random_ids = random.sample(range(1, 1000), 3)
+    return (
+        {
+            u'orgs': [{
+                u'key_id': u'{}'.format(str(i+1)),
+                u'organization_id': u'{}'.format(str(random_ids[i])),
+                u'organization': gen_string('alphanumeric'),
+            } for i in range(3)],
+            u'hcs': [{
+                u'key_id': u'{}'.format(str(i+1)),
+                u'org_id': u'{}'.format(str(random_ids[i])),
+                u'name': gen_string('alphanumeric'),
+            } for i in range(3)]
+
+        },
+    )
```

Comment:
How about something like this?

```python
def import_hc_data():
    """Random data for Organization Import tests"""
    org_ids = (type(u'')(org_id) for org_id in sample(range(1, 1000), 3))
    return ({
        u'orgs': [
            {
                u'key_id': type(u'')(i + 1),
                u'organization_id': org_ids[i],
                u'organization': gen_string('alphanumeric'),
            }
            for i in range(3)
        ],
        u'hcs': [
            {
                u'key_id': type(u'')(i + 1),
                u'org_id': org_ids[i],
                u'name': gen_string('alphanumeric'),
            }
            for i in range(3)
        ]
    },)
```

I've done a few things in the above:

* If you change `import random` to `from random import sample`, you can replace calls to `random.sample` with calls to `sample`.
* The `u'{}'.format(str(...))` bits work, but they can be shortened to `type(u'')(...)`.
* You can calculate all organization IDs and convert them to strings right up front. This removes some identical method calls from the code.
* You can drop some whitespace.
* You can compact some bits down to `({` and `},)`.

Thoughts?